mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py

```diff
@@ -176,17 +176,17 @@ def __call__(self, bboxes1, bboxes2, mode='iou', is_aligned=False):
 
         Args:
             bboxes1 (Tensor): shape (B, m, 6) in <x1, y1, z1, x2, y2, z2>
-            format or empty.
+                format or empty.
             bboxes2 (Tensor): shape (B, n, 6) in <x1, y1, z1, x2, y2, z2>
-            format or empty.
+                format or empty.
                 B indicates the batch dim, in shape (B1, B2, ..., Bn).
-                If ``is_aligned `` is ``True``, then m and n must be equal.
```

Comment:
**Space before the second "``"**
Before fix:
![image](https://user-images.githubusercontent.com/52420115/126042605-ccdd4a65-0043-4dd3-a98f-1069c23da17c.png)

After fix:
![Uploading image.png…]()